src/components/TokensPanel.vue

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
should be able to remove these ignores if everything is typed correctly

Reply:
Removed